src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb

```diff
@@ -12,9 +12,13 @@ Imports Microsoft.CodeAnalysis.Text
 Imports Roslyn.Test.Utilities
 Imports Roslyn.Test.Utilities.TestBase
 Imports Xunit
+Imports Microsoft.CodeAnalysis.Collections
 
 Friend Module CompilationUtils
 
+    Private ReadOnly _pooledStringBuilderPool As ObjectPool(Of PooledStringBuilder) = ParserTestUtilities.PooledStringBuilderPool
```

Comment:
>Private ReadOnly _pooledStringBuilderPool As ObjectPool(Of PooledStringBuilder) = ParserTestUtilities.PooledStringBuilderPool [](start = 4, length = 125)

This field is not needed. We could access ParserTestUtilities.PooledStringBuilderPool directly and, in fact, we don't have to use type name to qualify the name because the type is a Module.